creator/ingest_runs/queries/validation_run.py

```diff
@@ -24,14 +27,40 @@ class Query(object):
         filterset_class=ValidationRunFilter,
         description="Get all validation_runs",
     )
+    last_modified_active_validation_run = Field(
```

Comment:
I think this may be unnecessary. We can get the same behavior by adding additional fields to the `ValidationRunFilter` to allow selecting by a specific data review and set of states.